buildout.cfg

```diff
@@ -1,6 +1,6 @@
 [buildout]
 extends =
-    https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.x.cfg
+    https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.2.x.cfg
```

Comment:
This is not good. In `.travis.yml` we replace `test-5.x` with `test-$PLONE_VERSION.x.cfg`. With your change to 5.2.x, this replacement does not happen anymore, so we only test Plone 5.2 in all Travis jobs.

Reply:
sorry, this wasn't by intention - rather this is the result of a local tox run (sed is used there to generate the version). I'll revert that.

Reply:
on second thought, the buildout config created by sed could be named differently to prevent this. But that's maybe for another time.